### ISSUE TITLE
Doesn't work with Citra and v1.1 update installed

### ISSUE THREAD

**Issue by manuelx98** [leoetlino/project-restoration#107]
None of the three patches seems to work with Citra with either v1.0 .3ds or a decrypted v1.0 .cia (both EUR), having a v1.1 installed as .cia update. 
The emulator seems to totally ignore the existence of the patches if an update is present on a different location, and boots the game normally without the patch.
Trying to delete the v1.1 update and apply the v100 patch seems to work though.
Trying to use all three patches (in three different attempts) with the .app update makes the emulator crash.
My thoughts are that the emulator launches the update before the game, and therefore ignores the patch/es on the v1.0 game folder.
I didn't try a v1.1 dump since I only have a v1.0 cartridge.

**Comment by Lele99DD** [comment]
I'm not able to apply the patch either. As you said, Citra completely ignores the patch. It doesn't crash, it just loads the game normally, there's no sound effect telling me that the patch has been applied. But in my case even with the v1.0 game and the v100 patch it doesn't work. I've tried with older releases of the patch, different versions and older Citra releases, but nothing worked! This is how my game folder looks, just to be sure that I got it right:
![temp](https://user-images.githubusercontent.com/60327735/73141678-0e209380-4087-11ea-8014-6b2023eb9263.png)

**Comment by manuelx98** [comment]
> I'm not able to apply the patch either. As you said, Citra completely ignores the patch. It doesn't crash, it just loads the game normally, there's no sound effect telling me that the patch has been applied. But in my case even with the v1.0 game and the v100 patch it doesn't work. I've tried with older releases of the patch, different versions and older Citra releases, but nothing worked! This is how my game folder looks, just to be sure that I got it right:
> ![temp](https://user-images.githubusercontent.com/60327735/73141678-0e209380-4087-11ea-8014-6b2023eb9263.png)

I tried both my .3ds v1.0 and installed .cia v1.0 with the v100 patch without the v1.1 update installed and both are working with the patch. Check if you have the update installed (you can see by its icon in Citra or if right-clicking there's a "Open update data location") and if you are loading the game from there and not from another location. Also, you can try the other two patches if the v100 isn't working for you.

**Comment by Lele99DD** [comment]
I've tried again but unfortunately it didn't work. I've tried all possible combinations of files from the v100, v101 and v110 folders, closing and reopening Citra (Canary 1550) at every attempt and this didn't do anything, then I converted my .3ds game file into a .cia game file and installed this in Citra, but the game didn't show up in the list. I tried again to see if anything changed, but nothing did. Obviously these tests were done without any update installed, the game is in v1.0

**Comment by leoetlino** [comment]
@Lele99DD your exefsdir is misspelled ;)

**Comment by Lele99DD** [comment]
@leoetlino thanks, I didn't notice that. It works now, sorry for disturbing...

**Comment by manuelx98** [comment]
Any news?

**Comment by leoetlino** [comment]
Unfortunately I haven't had enough free time to look into this at all. I think you're correct -- it does sound like updates are causing Citra not to load patches for the original code at all. Have you tried checking with the Citra community (forum or Discord) to see if this is the intended behaviour?

**Comment by PurpleColonel** [comment]
Any luck with this? Any way to still play this? Past version that definitely works?

**Comment by manuelx98** [comment]
The issue now works a bit different but it's still there. Citra now officially supports mods, so you technically can right-click your game from Citra, select "Open mods location" and put your exheader.bin and /exefs/code.bps there, but, if you have the v1.1 update installed and the mod also installed, Citra will launch the game with the mod but without the v1.1 update.
In short, 
game v1.0 with the mod installed -> ok.
Game v1.0 + v1.1 update with the mod installed -> it launches it as v1.0 with mod.
Game v1.0 + v1.1 update without the mod installed -> it launches it as v1.1 (without the mod).
Basically it's the same problem but more user-friendly, I think?
So for now the only way to use the mod and the v1.1 update is to dump your game with v1.1 preinstalled and use the v110 mod, but people like me with v1.0 cartridge are screwed.

**Comment by leoetlino** [comment]
Yes, the mod will downgrade the game to 1.0 (temporarily).

Anyway, it seems that Citra's behaviour has changed and patches now override updates, so I think this issue can be considered to be resolved.
